misk-redis/src/main/kotlin/misk/redis/RedisClientMetrics.kt

```diff
@@ -2,10 +2,13 @@ package misk.redis
 
 import com.google.common.base.Stopwatch
 import com.google.common.base.Ticker
+import jakarta.inject.Inject
```

Comment:
javax?

Reply:
All the imports from the inject package seem to use `jakarta`, a search for `import javax.inject` returns nothing